Source/8_UI/Leaderboard/Components/Header/CaptorClan.cs

```diff
@@ -0,0 +1,149 @@
+using static AlphabetScrollInfo;
+
+namespace BeatLeader.Components {
+    internal class CaptorClan : ReeUIComponentV2 {
+        #region Components
+
+        [UIComponent("background"), UsedImplicitly]
+        private ImageView _background;
+
+        [UIComponent("text-component"), UsedImplicitly]
+        private TextMeshProUGUI _textComponent;
+
+        private bool _captorClanActive = false;
+
+        [UIValue("captor-clan-active"), UsedImplicitly]
+        private bool CaptorClanActive {
+            get => _captorClanActive;
+            set {
+                if (_captorClanActive.Equals(value)) return;
+                _captorClanActive = value;
+                NotifyPropertyChanged();
+            }
+        }
+
+        private bool _leaderboardCaptured = false;
+
+        [UIValue("leaderboard-captured"), UsedImplicitly]
+        private bool LeaderboardCaptured {
+            get => _leaderboardCaptured;
+            set {
+                if (_leaderboardCaptured.Equals(value)) return;
+                _leaderboardCaptured = value;
+                NotifyPropertyChanged();
+            }
+        }
+
+        private string _captorClanText = "";
+
+        [UIValue("captor-clan-text"), UsedImplicitly]
+        private string CaptorClanText {
+            get => _captorClanText;
+            set {
+                if (_captorClanText.Equals(value)) return;
+                _captorClanText = value;
+                NotifyPropertyChanged();
+            }
+        }
+
+        private string _captorClanHover = "";
+
+        [UIValue("captor-clan-hover"), UsedImplicitly]
+        private string CaptorClanHover {
+            get => _captorClanHover;
+            set {
+                if (_captorClanHover.Equals(value)) return;
+                _captorClanHover = value;
+                NotifyPropertyChanged();
+            }
+        }
+
+        private string _captorClanTextColor = "#FFFFFFDD";
+
+        [UIValue("captor-clan-text-color"), UsedImplicitly]
+        private string CaptorClanTextColor {
+            get => _captorClanTextColor;
+            set {
+                if (_captorClanTextColor.Equals(value)) return;
+                _captorClanTextColor = value;
+                NotifyPropertyChanged();
+            }
+        }
+
+        //private void InitializeText() {
+        //    _textComponent.enableAutoSizing = true;
+        //    _textComponent.fontSizeMin = 0.1f;
+        //    _textComponent.fontSizeMax = 2.0f;
+        //}
+
+        #endregion
+
+        #region Init/Dispose
+
+        protected override void OnInitialize() {
+            _background.material = BundleLoader.ClanTagBackgroundMaterial;
+        }
+
+        protected override void OnDispose() {
+        }
+
+        #endregion
+
+        #region SetValues
+
+        public void SetActive(bool value) {
+            // Used to be called on initialize
+            _background.material = BundleLoader.ClanTagBackgroundMaterial;

```

Comment:
why do you assign the material every time SetActive is called?

Reply:
Fixed; Happens in OnInitialize() now